### ISSUE TITLE
Add back plating to Top Sites

### ISSUE THREAD

**Issue by MozillaNoah** [mozilla-mobile/fenix#26606]
Add backplating UI according to the [Figma](https://www.figma.com/file/d7l1iGabQYqnLcnQbHDGzM/Wallpaper-2.0-%2B-Onboarding-v106?node-id=5536%3A189429). Existing functionality should remain.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21366)

**Comment by delia-pop** [comment]
Verified as fixed on the latest Nightly 106 from 09/06 and Debug build updated from main 1.0.2237 with Google Pixel 6 (Android 13).
The Top Sites are displayed as designed. However, we reported a minor issue related to an improper highlight on the Sponsored tiles: https://github.com/mozilla-mobile/fenix/issues/26854. 

![TopSites_Nightly106_Debug](https://user-images.githubusercontent.com/89388888/188909179-51cfcf38-b585-4123-8afc-4c7652022640.png)
